repo.yml

```diff
@@ -0,0 +1,4 @@
+type: 'node'
+upstream:
+  - repo: 'pinejs-client-core'
```

Comment:
I intentionally didn't add that, since this is a dev dependency in the API and though it would be better to avoid having the nested changelogs explode our deploy release notes.
Not against it, since in general we do want these.